scripts/eval/yamls/dbrx-gauntlet/dbrx-18b.yaml

```diff
@@ -0,0 +1,72 @@
+# Eval pre-trained sizes of DBRX models. See these docs for more information
+# about pre-training your own DBRX model:
+# https://docs.mosaicml.com/projects/mcli/en/latest/pretraining/pretraining.html#supported-models
+
+# Specify icl_tasks and or eval_gauntlet to run
+eval_gauntlet: eval/yamls/eval_gauntlet_v0.3.yaml
+icl_tasks: eval/yamls/tasks_v0.3.yaml
+
+models:
+- model_name: dbrx-18b
+  load_path: s3://bucket/checkpoints/dbrx-9b/checkpoint-1000000  # TODO: Update with correct save folder
+  tokenizer:
```

Comment:
tokenizer should be the dbrx tokenizer

Reply:
Switching to tiktoken/gpt-4. 

```
tokenizer:
  name: tiktoken
  kwargs:
    token: true
    name: gpt-4
    eos_token_id: 100257 
```

Reply:
Whoops eos_token_id isn't in the tokenizer but in train_loader.dataset and eval_loader.dataset.